### DISCUSSION TITLE
Preserve scroll position when visiting a subreddit or post

### DISCUSSION THREAD

**Discussion by shreya-shah** [Docile-Alligator/Infinity-For-Reddit#1570] [Ideas]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/ideas) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/completed-ideas) ideas to see if someone else has already submitted the same request.
- [X] I will describe the feature with as much detail as possible.

### Description

Need a feature to preserve scroll position when visiting a subreddit or viewing comments so that we don't need to scroll again on revisiting immediately.

### Additional context

This would be a good to have feature as it helps when there are a lot of comments which can require repeated scrolling if we have gone back by accident. Will be really helpful in quickly going back to viewing/reading on something.

### How important is this feature?

Nice to have

**Comment by Gtt1229** [comment]
@Docile-Alligator Any input on what needs to be implemented to get this rolling?
